### PR TITLE
add params_description space to nextflow.config

### PR DESCRIPTION
**Pull request by KochTobi**
Introduce `params_description` space to the config.
Add default location.

### CODE REVIEW

File: nextflow.config [hunk 1]
```diff
@@ -12,7 +12,7 @@
 params {
 
   container = 'nfcore/hlatyping:1.1.4' // Container slug. Stable releases should specify release tag!
-  
+
```
Comment (sven1103) [inline nextflow.config]:
```suggestion
```

**Comment by sven1103** [comment]
contributes to #57

**Comment by sven1103** [comment]
and to nf-core/tools#267

**Comment by ewels** [comment]
tabs

😱

**Pull request merged**